src/components/header/index.css

```diff
@@ -558,6 +558,19 @@
   margin: 0;
 }
 
+li {
```

Comment:
This is for all li? That's not good to do here.

Reply:
That just gives spaces between the list items. Shall I use `line-height` instead?

Reply:
The problem is that you're setting the css for ALL list elements, no?

Reply:
Oh okay. I'll make sure it sets the styles for these particular list item only.